imap-to-local-html.py

```diff
@@ -335,6 +335,29 @@ def getMailFolders():
             if mailFolders[menu]["parent"] == menusWithNoParent[0]:
                 mailFolders[menu]["parent"] = ""
 
+    # Add in any additional folders that may be mbox only
+    for i in server.get('folders'):
+        exists = False
+        for e in mailFolders:
+            if mailFolders[e].title == i:
+                exists = True
+
+        if exists = True:
+            continue
+
+        count += 1
+
+        fileName = "%03d-%s.html" % (count, slugify_safe(normalize(i, "utf7"), defaultVal="folder"))
+
+        mailFolders[i] = {
+            "id": i,
+            "title": i,
+            "parent": "",
```

Comment:
All folders in such a case are 1-level only?

Reply:
Yes, this would only cope with single level mbox folders as I havent any further depth to test against and iCloud doesnt support nested folders for me to test